karma.conf.js

```diff
@@ -96,6 +89,6 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: true
+    singleRun: false
```

Comment:
No single run by default, so that I could start a local, autowatching, persistent karma using `$ karma start`.

The single run in the CI is caused by an option in the `package.json`.